apps/pie-docs/src/components/switch/overview/overview.md

```diff
@@ -0,0 +1,298 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Switch
+    order: 1
+shouldShowContents: true
+---
+
+## Overview
+The purpose of switches is to offer users a quick and straightforward method to switch between two distinct options or states. Switches enhance the user experience by providing a clear and efficient way to make selections or enable/disable functionality.
+
+Switches are commonly used to enable or disable settings, switch between modes or perform binary actions in user interfaces. When users interact with switches, they change their appearance to indicate the current state or selection.
```

Comment:
Perhaps a recommendation of what to use when there are more than 2 states (delivery/pickup/dine-in could be one example)

Reply:
Added a comment to the ticket, so Designers can check it during Design review.